_schemaV2.graphql

```diff
@@ -12059,6 +12059,8 @@ type MarketingCollection {
   category: String!
   createdAt: ISO8601DateTime!
   credit: String
+
+  # HTML description, derived from the markdown description
   description: String
```

Comment:
Most of these are just "note" metadata updates 

The only real details is the removal of `PartnerLocation` on Artwork Type!